### ISSUE TITLE
That random startup crash

### ISSUE THREAD

**Issue by TeoTwawki** [DarkstarProject/xiloader#17]
Sometimes at account login, xiloader will crash on receiving the character list. Can immediately close and relaunch without issue but this is frequent and random and somehow in 4+ yrs nobody has raised the issue, so here it is. On the lobby side of things everything looks normal.

![image](https://user-images.githubusercontent.com/6871475/36953840-a7ab7d1c-1feb-11e8-8aa8-db7519361253.png)

We also have a random crash that can occur on alt tabbing out/back in. No idea if its related.

**Comment by teschnei** [comment]
I have never seen this happen, actually.  xiloader only ever "crashes" for
me when I am closing it.

Is this without a windower?

**Comment by TeoTwawki** [comment]
yeah. Maybe hairpin related? I use hairpin often. its faster for me to make 1 laptop build and run the game client off the other, on the same lan.

**Comment by TeoTwawki** [comment]
actually can't be all hairpins fault, because I've had the same crash logging into other servers than my local copy and I don't use hairpin for that :thinking:
